### PR TITLE
mapper,beta deprication, and proxy removed

### PR DESCRIPTION
**Pull request by eyalpalo**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-1844)

## Description
Made changes in pop3 integration to remove it from beta version.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MailListener_-_POP3/Integrations/MailListener_POP3/MailListener_POP3.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import json
+
```
Comment (dantavori) [inline Packs/MailListener_-_POP3/Integrations/MailListener_POP3/MailListener_POP3.py:2]:
```suggestion
```
in CSP already

File: Packs/MailListener_-_POP3/Integrations/MailListener_POP3/MailListener_POP3.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 category: Messaging
 commonfields:
-  id: MailListener - POP3 Beta
+  id: MailListener - POP3
```
Comment (dantavori) [inline Packs/MailListener_-_POP3/Integrations/MailListener_POP3/MailListener_POP3.yml:3]: not sure we can modify integration id and names. if not - change only display and we'll live with that

**Review by dantavori — CHANGES_REQUESTED** [review]
looks good, see notes below

**Review by dantavori — APPROVED** [review]

File: Packs/MailListener_-_POP3/Integrations/MailListener_POP3/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+## Overview
```
Comment (dantavori) [inline Packs/MailListener_-_POP3/Integrations/MailListener_POP3/README.md:1]: how was this generated? it doesn't look like the regular one

File: Packs/MailListener_-_POP3/ReleaseNotes/2_0_0.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+##### MailListener - POP3
+- Removed integration from beta version.
+- Added **Incoming mapper** to map incoming emails to **Phishing** incident type.
```
Comment (dantavori) [inline Packs/MailListener_-_POP3/ReleaseNotes/2_0_0.md:5]:
```suggestion
- Added a default *incoming mapper* for mapping incoming emails to the *Phishing* incident type.
```

**Comment by eyalpalo** [comment]
@ShahafBenYakir Force merge is needed here. Due to the reason we removed one of the integration parameters, which caused one of the validations to fail.

**Pull request merged**